tests/commands/base/test_paths.py

```diff
@@ -9,6 +10,39 @@
 from .conftest import DummyCommand
 
 
+def test_path_is_realpath(tmp_path):
+    """Briefcase's data path matches its realpath.
+
+    For the Windows Store Python, filesystem interaction with
+    `%LOCALAPPDATA%` can be redirected to a sandboxed location. However,
+    `os.path.realpath()` will reveal such a redirection.
+    """
+    # %LOCALAPPDATA% seemingly isn't available in GH CI...
```

Comment:
Is that an indicator of a difference between Windows 11 vs Windows Server? Is that something we need to accomodate more generally?

Reply:
Or - will it cause failures *outside* GH CI?

Reply:
Tracked it down; `tox` strips many environment variables from the environment the tests run in. I explicitly passed it through for the unit tests.